### PR TITLE
Add support for `tfvars` files

### LINKED ISSUE #426: Please add support for formatting .tfvars files
The old version of the Terraform extension formatted `.tfvars` files so why doesn't the new version?  Please restore this feature.

Thanks

### PR DESCRIPTION
**Pull request by beandrad**
To add completion/hover/highlighting on tfvars (variable files).

Fixes #426

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,3482 @@
 {
   "name": "terraform",
   "version": "2.12.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (radeksimko) [inline package-lock.json:4]: Happy to upgrade - Just for posterity - which version of NPM do you have installed locally?

I wish there was a way of pinning it, like Node version via https://github.com/hashicorp/vscode-terraform/blob/main/.nvmrc 🤔

Comment (beandrad) [reply package-lock.json:4]: I have version 7.6.3, a bit old 🙈

Comment (radeksimko) [reply package-lock.json:4]: which is still newer than mine (6.14.8) 😂  🙈 

... which explains the `lockfileVersion` bump I guess ...

**Review by radeksimko — APPROVED** [review]
LGTM, thanks!

**Pull request merged**